public/scripts/controllers/playController.js

```diff
@@ -0,0 +1,122 @@
+angular.module('scrambleApp').
+  controller('playController', ['$scope', '$interval', 'wordFactory', 'gameStateService',
+             function($scope, $interval, wordFactory, gameStateService) {
+    //Controls the gameplay part
+
+    $scope.letters = [];
+    $scope.userInput = [];
+    $scope.score = 0;
+
+    var inputReady = true;
+    var currentWord;
+    var numCharacters; //number of characters
+    var startTime; //marks when word was given to give score
+    var endTime = Date.now() + 60999; //set end time to 60s from current time
+    var timer;
+
+    //Set scope 'letters' to a new random word
+    var getRandomWord = function() {
+      inputReady = false;
+      wordFactory.getRandomWord(function(randomWord, error) {
+        if (error) {
+          console.log(error);
+        } else {
+          currentWord = randomWord;
+          var shuffledWord = wordFactory.shuffleWord(currentWord);
+          $scope.letters = shuffledWord.map(function(letter) {return {letter:letter}});
```

Comment:
not sure why you need to do this map to return an object that has a key:value letter to something, it looks like you can simply use an array to do the ng-if and do a track by $index